website/docs/r/linux_function_app.html.markdown

```diff
@@ -291,11 +291,9 @@ An `active_directory_v2` block supports the following:
 
 * `client_secret_setting_name` - (Optional) The App Setting name that contains the client secret of the Client.
 
-!> **NOTE:** A setting with this name must exist in `app_settings` to function correctly.
```

Comment:
This needs to remain I think. `clientSecretCertificateThumbprint` is not an app service app_setting value afaik, so does not need the note changing to the below, it can simply be removed. This will apply to all the docs changed in this way.